tools/SiblingKiller.py

```diff
@@ -89,7 +91,7 @@ def kill(self, pid=0, pids=list()):
 
     def siblings(self):
         my_name = self.my_process_name()
-        return filter(lambda x: x < self.__my_pid, 
+        return filter(lambda x: x < self.__my_pid,
```

Comment:
They make up a queue. In order not to have all siblings kill each other so that none is left, I kill only those whose id's are lower than "mine", thus we can be sure that one will be left. There were situations in the past when the server couldn't start because SiblingKillers all killed each other.